### ISSUE TITLE
Unable to install on Mac M1

### ISSUE THREAD

**Issue by juanbigotes** [AI-sandbox/neural-admixture#14]
Hi,

I was trying to install the requirements but torch is getting me a hard time on Mac M1. Torch will only run on arm64 from v1.9 and above. However, when I try to install neural-admixture via pip it won't allow to use any other version of torch but 1.7.1. Is there any way to allow the use of a higher version of torch?

Thank you!

**Comment by AlbertDominguez** [comment]
Hi @juanbigotes,

This is something we already have noticed. We were planning to release a hotfix on the next update along with another issue resolution later this weekend.

For the moment, you can manually solve it by cloning the repository and changing the `install_requires` field in the `setup.py` file (Line 20 in latest release) to
```python
install_requires=['codetiming==1.3.0',
                      'h5py>=3.1.0',
                      'matplotlib==3.3.4',
                      'numpy>=1.21.0',
                      'pandas==1.2.4',
                      'pandas_plink==2.2.9',
                      'py_pcha==0.1.3',
                      'scikit-allel==1.3.5',
                      'scikit-learn>=0.24.1',
                      'setuptools==50.3.1',
                      'torch>=1.7.1',
                      'wandb==0.10.21']
```


However, the installation of h5py using this still breaks. Having looked around a bit, I recommend you install `h5py` before installing `neural-admixture` using conda. The steps would look like
```bash
> (nadm) cd neural-admixture
> (nadm) conda install h5py
> (nadm) pip install -e .
```

We'll let you know when we release the hotfix so the package could be directly installed from PyPi (`pip install neural-admixture`), but installing h5py from conda would be required nevertheless for the moment.

Thank you,
Albert

**Comment by AlbertDominguez** [comment]
Fixed in v1.1.4.
